client/src/views/SearchPage.vue

```diff
@@ -1,27 +1,22 @@
 <template>
     <div>
-        <br><br>
-        <p>You searched (Country, City)<br>{{this.$route.params.country}}<br>{{this.$route.params.city}}</p>
-        <br>
-        <p v-if="budget != 0">With a budget of ${{budget}}</p>
-        <br><br>
-
-        <v-btn
-            elevation="2"
-            @click="$router.push('/')"
-        >
-            Back
-        </v-btn>
+        <MapComponent/>
     </div>
 </template>
 
 <script>
+
+import MapComponent from "../components/MapComponent"
+
 export default {
     data: () => {
         return {
-            budget: 0
+            budget: 0,
         }
     },
+    components: {
+        MapComponent
+    },
     async mounted() {
```

Comment:
Why is the mounted function async. In general, it is best to avoid blocking/asynchronous in lifecycle hooks